readme.md

```diff
@@ -16,35 +16,49 @@ $ npm install --save hook-std
 const assert = require('assert');
 const hookStd = require('hook-std');
 
-const unhook = hookStd.stdout(output => {
+const promise = hookStd.stdout(output => {
+	promise.unhook();
+
+	assert.strictEqual(output.trim(), 'unicorn');
+});
+
+console.log('unicorn');
+await promise;
+```
+
+Unhook can also be done via callback:
+
+```js
+const promise = hookStd.stdout((output, encoding, unhook) => {
 	unhook();
 
 	assert.strictEqual(output.trim(), 'unicorn');
 });
 
 console.log('unicorn');
+await promise;
 ```
 
 
 ## API
 
-### hookStd([options], callback)
+### hookStd([options], transform)
 
 Hook stdout and stderr.
 
-Returns a function that unhooks stdout and stderr.
+Returns a `Promise` that unhooks stdout and stderr.
```

Comment:
This is not entirely correct. It returns a Promise that resolves when it's unhooked, and with a `.unhook()` method.